src/TriggerBinding/SqlTriggerConstants.cs

```diff
@@ -10,5 +10,9 @@ internal static class SqlTriggerConstants
         public const string GlobalStateTableName = "[" + SchemaName + "].[GlobalState]";
 
         public const string WorkerTableNameFormat = "[" + SchemaName + "].[Worker_{0}]";
+
+        public const string WorkerTableChangeVersionColumnName = "ChangeVersion";
```

Comment:
I like the idea of prefixing these with `_az_func_` - since this isn't actually published yet there shouldn't be any issues with renaming columns.

Reply:
Also looking for opinion on `AzFuncAttemptCount`.

Reply:
What do you mean? I think prefixing all the internal column names with `_az_func_` is a good idea. (so it'd be `_az_func_attemptcount`)